src/WebAppDIRAC/Lib/WebHandler.py

```diff
@@ -75,18 +75,20 @@ def web_myMethod(self):
         return FileResponse(data, 'filename', 'csv')
     """
 
-    def __init__(self, payload=None, fileName=None, ext=None, cache=True):
+    def __init__(self, payload=None, fileName: str = None, ext: str = None, cache: bool = True):
```

Comment:
This is the wrong type hint, it should be:
```suggestion
    def __init__(self, payload=None, fileName: Optional[str] = None, ext: Optional[str] = None, cache: bool = True):
```
Where `Optional` comes from `from typing import Optional`. In Python 3.10+ it can be `None | str`.

Reply:
Good point, fixed